packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -107,7 +108,8 @@ - (instancetype)initWithMapView:(GMSMapView *_Nonnull)mapView
     _registrar = registrar;
     _markersController = [[FLTMarkersController alloc] initWithMethodChannel:_channel
                                                                      mapView:_mapView
-                                                                   registrar:registrar];
+                                                                   registrar:registrar
+                                                           cozyMarkerBuilder:[[CozyMarkerBuilder alloc] initCozy]];
```

Comment:
the marker builder (together with the caching store) will be instantiated as soon as the map is instantiated if you change cities, change screens the same caching will persist (in the java codebase it behaves the same way)